src/Mutable.php

```diff
@@ -146,7 +146,7 @@ public function generate()
             }
 
             //TODO: handle whitespace!
-            if (is_array($token) && $token[0] == T_FUNCTION) {
+            if (is_array($token) && $token[0] == T_FUNCTION && $className !== '???') {
```

Comment:
It's a potential mistake I probably never made. It's not worth the readability tradeoff IMO.